### ISSUE TITLE
OpenVPN client credentials cannot be renewed or deleted

### ISSUE THREAD

**Issue by sbruzzese902** [sighupio/furyagent#32]
Currently every time client credentials expire we have to create new credentials with:
- no possibility to delete the old ones
- no possibility to renew the old ones or use the same name as existing credentials

In the long term this leads to the spawning of a lot of useless credentials ( number of users X number of years ).

I don't know which is the preferred behaviour (delete or renew) but any of them would be really useful.

**Comment by lzecca78** [comment]
Yes, this will help a lot the day-by-day activities and is a very nice `good first issue`.
I guess that the `culprit`  is the function `CreateUser` that should take the parameter `overwrite` in order to force the recreation/upload of the certificate with the same name.
